google/cloud/spanner_dbapi/connection.py

```diff
@@ -74,6 +103,11 @@ def run_prior_DDL_statements(self):
         return self.__handle_update_ddl(ddl_statements)
 
     def list_tables(self):
+        """Lists all existing tables.
+
+        :rtype: str
+        :returns: tables with corresponding information.
```

Comment:
This method is supposed to return a `list` not a string, of the names of the available tables and views. Take a closer look into the underlying logic and see how this method is linked with `django_spanner` functionality.